server.py

```diff
@@ -129,9 +129,9 @@ def get_rating_():
     url = req_data["url"]
     res = ratings_collection.find_one({"url": url})
     if res:
-        response = jsonify({"rating": res["rating"]})
+        response = jsonify({"rating": res["rating"], "freq": res["freq"]})
     else:
-        response = jsonify({"rating": 0})
+        response = jsonify({"rating": 0, "freq": 0})

```

Comment:
This is good for a start, to count the number of ratings by users. But the problem it will have is that a single user could do 10 ratings for a webpage and it will count all of them.

Reply:
I agree, maybe we could restrict the number of reviews to one per user per page? Please let me know what you think.

Reply:
Since this is outside the scope of this issue, I will submit a PR for now and create another issue to restrict the number of reviews per user. Thank you!

Reply:
Since the required change would be out of the scope of this issue, will submit a PR with this and create another issue for restricting the number of reviews per user.